bigtable/client/readrowsparser.h

```diff
@@ -0,0 +1,121 @@
+// Copyright 2017 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef BIGTABLE_CLIENT_READROWSPARSER_H_
+#define BIGTABLE_CLIENT_READROWSPARSER_H_
+
+#include "bigtable/client/version.h"
+
+#include "bigtable/client/cell.h"
+#include "bigtable/client/row.h"
+
+#include "google/bigtable/v2/bigtable.grpc.pb.h"
+
+#include <absl/strings/string_view.h>
+#include <vector>
+
+namespace bigtable {
+inline namespace BIGTABLE_CLIENT_NS {
+/**
+ * The internal module responsible for transforming ReadRowsResponse
+ * protobufs into Row objects.
+ *
+ * Users are expected to do something like:
+ *   while (!stream.EOT()) {
```

Comment:
Insert a blank line above this, and then this on a line by itself:

```
@code
```

Reply:
done